packages/wonder-blocks-clickable/src/components/clickable-behavior.ts

```diff
@@ -560,21 +556,21 @@ export default class ClickableBehavior extends React.Component<
         if (onKeyDown) {
             onKeyDown(e);
         }
-
-        const keyCode = e.which || e.keyCode;
+        // Allow tests to use mixed case commands ("Space" or "space")
```

Comment:
It seems interesting to modify the component logic to accommodate the casing used in tests!

It looks like [user-event will also use KeyboardEvent.key](https://testing-library.com/docs/user-event/keyboard) (or KeyboardEvent.code). I wonder if it's enough to solve the issue in tests by using the `keys` constant as well so we're testing with the proper keys that the browser uses (which you've updated already!). Let me know what you think! I'm still learning more about user-event so let me know if I am missing something!

Reply:
Added a bunch of details in my previous comment!